ts/DataGrid/Options.ts

```diff
@@ -391,6 +399,34 @@ export interface HeaderEvents {
     click?: ColumnEventCallback;
 }
 
+export interface GroupedHeader {
```

Comment:
I would name it `GroupedHeaderOptions`. I mean, the `Options` keyword is important here.